client/app/pod/popular.service.js

```diff
@@ -0,0 +1,39 @@
+angular.module('throwcast.popular')
```

Comment:
I'm leaning towards not making a pop service. The pop controller probably should just call out to two separate services. The PodcastService and the StationService. Unless you have a good reason for making the PopularService.